### ISSUE TITLE
"You have no new notifications" 

### ISSUE THREAD

**Issue by Bon18** [AllanWang/Frost-for-Facebook#1762]
- [x] I have verified that the bug does not occur in my mobile browser (at https://touch.facebook.com/, https://m.facebook.com) and other apps found in Play store or F-Droid. 

**Describe the bug**
Whenever I tap on "Notifications" it says "you have no new notifications". After refreshing the notification page several times (irregular) I can see the Facebook notifications. I have system notifications turned off in Frost, since I don't need. But, I need to access the Facebook notifications page. This error occurs everytime, even if I have new notifications. And it should display all notifications, even if there's no new. Several times a day, I need to access old notifications. 

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'Notifications'
2. Click on 'Any notification from posts, comments, etc.'
3. Tap on the Notifications again on the Nav bar

**Details (please provide at least the app version):**
 - App Version: 3.0.0
 - Device: Samsung Galaxy J7 Pro. (J730GM). 
 - Android Version: 9 (Pie). 

Screenshots: 
! [Screenshot_20210306-215205_Frost](https://user-images.githubusercontent.com/80167060/110213806-a5315f00-7ec7-11eb-9d0c-d1476b988b5e.jpg)
! [20210306_221617](https://user-images.githubusercontent.com/80167060/110214255-c72be100-7ec9-11eb-8f78-c75e1bde1da1.jpg)

**Comment by AllanWang** [comment]
Does it also happen if you pull down to refresh? Can you confirm again in your browser, since Frost doesn't really do anything special when loading that page. 

The only potential issue I can think of is the user agent, but anything else, even if reproducible, probably can't be fixed.

**Comment by Bon18** [comment]
With due respect to the Developer and contributors,

First, I must say that the app is excellent. And yes, the issue occurs
everytime I use Notifications. Even after several pull-down to refresh, it
remains same: "You have no new notifications". But none of the browsers do
this. I checked another app available in F-Droid that is SlimSocial, and
that app also worked fine. But SlimSocial just looks like the browser and
functions like a browser (UI, performance), so I decided to stick with
Frost, in the hope for a fix.

The issue occurs in two ways:
1. When I tap on Notifications (even after a day, when I definitely have at
least 20 new notifications).
2. When the first attempt showed all new notifications, but after clicking
on one, - then returning to the Notifications tab again. To see other
unread or "un-clicked" notifications, I need to pull-down several times.
Highest I got 25 times pull-down. And sometimes I need to close the app
from Android Recent apps list, then Relaunch to make it work.

I confirm that this issue does not occur in any browser (mobile or desktop
mode), any other apps, or android system web-view mode. After receiving
this mail, I switched different user agent in android Firefox browser (with
the help of user agent switcher extension), and found no problem.

Hope to see the improvement soon.
❤️

Sincerely,
S. K. Biswas
